sentry-logback/src/main/java/io/sentry/logback/SentryAppender.java

```diff
@@ -0,0 +1,164 @@
+package io.sentry.logback;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.classic.spi.ThrowableProxy;
+import ch.qos.logback.core.UnsynchronizedAppenderBase;
+import io.sentry.core.Sentry;
+import io.sentry.core.SentryEvent;
+import io.sentry.core.SentryLevel;
+import io.sentry.core.protocol.Message;
+import java.util.*;
+import java.util.stream.Collectors;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/** Appender for logback in charge of sending the logged events to a Sentry server. */
+public final class SentryAppender extends UnsynchronizedAppenderBase<ILoggingEvent> {
+  private @Nullable String dsn;
+  private @Nullable String environment;
+  private @Nullable Integer maxBreadcrumbs;
+  private @Nullable Integer shutdownTimeoutMillis;
+  private @Nullable Integer flushTimeoutMillis;
+  private @Nullable Integer readTimeoutMillis;
+  private @Nullable Double sampleRate;
+  private @Nullable Boolean bypassSecurity;
+  private @Nullable Boolean debug;
+  private @Nullable Boolean attachThreads;
+  private @Nullable Boolean attachStacktrace;
+
+  @Override
+  public void start() {
+    if (dsn != null) {
+      Sentry.init(
+          options -> {
+            options.setDsn(dsn);
```

Comment:
@marandaneto @bruno-garcia should we set here custom `sentryClientName` or do we stick to one set by `sentry-core`?

Reply:
yes I agree, having the granularity data would be nice